TwilioVerifyDemo/TwilioVerifyDemo/FactorDetail/Presenter/FactorDetailPresenter.swift

```diff
@@ -39,16 +39,11 @@ class FactorDetailPresenter {
   }
   
   init?(withView view: FactorDetailView?, factor: Factor) {
```

Comment:
ditto

Reply:
Added a validation for the twilio verify dependency